lib/new_utils.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * @param {String} entity
+ * @param {String} name
+ */
+function markAsNotSupported(entity, name) {
+  throw new ReferenceError(
+    "The ${entity} (${name}) is not supported by our driver",
+  );
+}
```

Comment:
Now the semantics of this function would be better described by the name: `throw[Error]NotSupported[Entity]`, where chunks in brackets are optional IMO.